packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -294,7 +294,7 @@ const OpenPositions = ({
     const [active_index, setActiveIndex] = React.useState(is_multiplier ? 1 : 0);
     // Tabs should be visible only when there is at least one active multiplier contract
     const [has_multiplier_contract, setMultiplierContract] = React.useState(false);
-
+    const [active_positions_saved, set_active_positions] = React.useState([]);
```

Comment:
Can we call keep the names for the variable and function consistent, f.e. `active_positions_saved` => `setActivePositionsSaved`?